src/browser/BrowserSettings.cpp

```diff
@@ -514,3 +517,13 @@ void BrowserSettings::updateBinaryPaths()
 {
     m_nativeMessageInstaller.updateBinaryPaths();
 }
+
+QString BrowserSettings::replaceHomePath(QString location)
+{
+    auto homePath = QDir::homePath();
+    if (location.startsWith(homePath)) {
+        location.replace(homePath, "~");
+    }
+
+    return location;
+}
```

Comment:
You are missing the ~ to absolute dir code branch. The ~ path is stored in config and used in the browser code. This is the opposite of your description. This should also only occur on Linux systems.

Reply:
Hmm? Gotta double check this one.

Reply:
Fixed the brain fart.